src/config.rs

```diff
@@ -375,16 +375,26 @@ pub fn parse_style_string(style_string: &str) -> Option<ansi_term::Style> {
                     "bold" => Some(style.bold()),
                     "italic" => Some(style.italic()),
                     "dimmed" => Some(style.dimmed()),
-                    "none" => None,
-
-                    // Try to see if this token parses as a valid color string
-                    color_string => parse_color_string(color_string).map(|ansi_color| {
-                        if col_fg {
-                            style.fg(ansi_color)
+                    color_string => {
+                        if color_string == "none" && col_fg {
+                            None
                         } else {
-                            style.on(ansi_color)
+                            let parsed = parse_color_string(color_string);
```

Comment:
I find this block a little hard to understand on quick scan. Could you add a comment here (briefly) explaining what the goal here is?

Reply:
Here we decide what to do with the string when it is supposed to be a color. Either yield a None style, reset background color or set the appropriate fg/bg color. Comments added in the code.